conformance/packages/dns-test/src/tshark.rs

```diff
@@ -112,20 +112,6 @@ impl Tshark {
         Ok(())
     }
 
-    /// Blocks until `tshark` reports the number of expected captured packets.
-    pub fn wait_for_new_packets(&mut self, expected: usize) -> Result<usize> {
```

Comment:
Suggest squashing this with the previous commit to make it more obvious what this is being replaced with.